core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorBodyDirective.js

```diff
@@ -28,10 +28,27 @@ oppia.directive('simpleEditorBody', [
         '$scope', 'SimpleEditorManagerService',
         'explorationSaveService', 'explorationRightsService',
         'explorationWarningsService', 'QuestionIdService',
-        function($scope, SimpleEditorManagerService,
+        'StatesToQuestionsService', function($scope, SimpleEditorManagerService,
             explorationSaveService, explorationRightsService,
-            explorationWarningsService, QuestionIdService) {
+            explorationWarningsService, QuestionIdService,
+            StatesToQuestionsService) {
           $scope.data = SimpleEditorManagerService.getData();
+          $scope.chooseQuestionType = false;
+          $scope.modalWrapperVisible = false;
```

Comment:
Please take a look at the sample dropdowns on the Bootstrap page; they include the functionality you mention. If you can use that dropdown, you probably don't even need a variable to track open/closed state.